frontend/main.js

```diff
@@ -0,0 +1,51 @@
+async function getTodos(url){
+    return await fetch(url)
+    .then((response) => {
+        if(response.status === 200){
+            return response.json()
+        }
+        throw new Error(`${response.status} - ${response.statusText}`);
+    })
+};
+
+getTodos("todos.json")
+.then((response) => {
+    if(!response.length){
+        // Add "how to create a new todo" instructions when empty
+    }else{
+        for(let todo of response){
+            displayToDo(todo);
+        }
+        searchBoxFunction(response);
+    }
```

Comment:
Whenever you see an `else`, you can, for sure, refactor it into something more simple to read/maintain

```suggestion
    if(!response.length){
        // Add "how to create a new todo" instructions when empty
        return;
    }
    
    for(const todo of response) {
        displayToDo(todo);
    }
    
    searchBoxFunction(response);
```

Remember to always start your variables as `const` and then change them to `let `only if necessary.